### DISCUSSION TITLE
Zapret для Proxmox

### DISCUSSION THREAD

**Discussion by almirus** [bol-van/zapret#197] [Q&A] [answered]
Приветствую!
Какой инструкцией пользоваться чтобы на домашнем MiniPC с Proxmox поднять Zapret для обхода на домашних устройствах замедление Youtube (особенно Android TV)

**Comment by bol-van** [comment]
Если надо только замедление обойти, то используем nfqws --dpi-desync=split2 с hostlist фильтром. В /opt/zapret/ipset/zapret-hosts-user.txt пишем googlevideo.com
plain http отключаем, оставляем https
QUIC: --dpi-desync=fake   , если он полностью не блокируется провайдером. если блокируется - можно не включать

Пошаговых инструкций в стиле "нажмите тут" я не пишу, но есть docs/quick_start.txt. там вкратце описан общий процесс установки zapret

**Comment by almirus** [comment]
@bol-van спасибо за ответ. но если сервис будет запущен на Proxmox *на отдельном PC), обход же будет работать только для хост машины, остальные машины в сети не пойдут через него в сеть или он работает как прокси?

**Comment by bol-van** [comment, marked as answer]
Основной режим работы zapret - это работа на шлюзе.
Если для него используется другой хост, то получается остается только прокси.
Можно выбрать режим MODE=tpws-socks и параметр tpws   --split-tls=sni
Или если все-же хотите nfqws или нужен HTTP proxy вместо socks, то можно подвесить отдельно проксю. squid, допустим

**Reply by atheisthax** [reply]
А подскажите, если я установлю zapret на openwrt в виртуалке, а на основном шлюзе перенаправлю запросы на ютуб и прочее на это виртуалку - у меня будет работать?

**Reply by DimaXA97** [reply]
> А подскажите, если я установлю zapret на openwrt в виртуалке, а на основном шлюзе перенаправлю запросы на ютуб и прочее на это виртуалку - у меня будет работать?

Отвечу - да, работать будет. 
Однако, нужно решить проблему "петли".
Если на основном роутере добавлять маршруты к ют/инст/etc через виртуалку, то нужно добавить правило с отдельной таблицей маршрутизации для этой виртуалки, где 0.0.0.0/0 via internet GW, иначе трафик попадет в петлю и все сломается.
